make/compiler/Makefile.clang

```diff
@@ -77,6 +77,10 @@ SPECIALIZE_CFLAGS = -march=$(CHPL_MAKE_TARGET_ARCH)
 endif
 endif
 
+# Set flag for lax or IEEE floating point
+NO_IEEE_FLOAT_GEN_CFLAGS = -ffast-math
+IEEE_FLOAT_GEN_CFLAGS =
```

Comment:
Could throw `-fno-fast-math` for completeness though it won't have any functional impact